pkg/inference/preset-inferences.go

```diff
@@ -0,0 +1,167 @@
+package inference
+
+import (
+	"context"
+	"fmt"
+
+	kdmv1alpha1 "github.com/kdm/api/v1alpha1"
+	"github.com/kdm/pkg/k8sresources"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	PresetSetModelllama2AImage = "aimodelsregistry.azurecr.io/llama-2-7b-chat:latest"
+	PresetSetModelllama2BImage = "aimodelsregistry.azurecr.io/llama-2-13b-chat:latest"
+	PresetSetModelllama2CImage = "aimodelsregistry.azurecr.io/llama-2-70b-chat:latest"
```

Comment:
I see these are all for chat - eventually we'll also add the text completion images these are just without -chat suffix

Reply:
Addressed.